### ISSUE TITLE
Default value set in custom operation doesn't get stored

### ISSUE THREAD

**Issue by danybol** [directus/directus#23001]
### Describe the Bug

When creating a custom operation, you can set default values for the options. These defaults are correctly populated in the creation form when you add the operation to a flow. However, when the operation has been added to the flow the option values are undefined.

### To Reproduce

1. Create a new custom operation with an option that includes a default value, e.g.:
```
import { defineOperationApp } from '@directus/extensions-sdk'

export default defineOperationApp({
	id: 'some-operation',
	name: 'Some operation',
	icon: 'box',
	description: '',
	overview: ({ maxPages }) => [
		{
			label: 'Max pages to return',
			text: `${maxPages}`,
		}
	],
	options: [
		{
			field: 'maxPages',
			name: 'Max number of pages to return',
			type: 'number',
			schema: {
				default_value: 1000
			}
		}
	],
})
```

2. Add this operation to a flow. The default is correctly filled in:
![image](https://github.com/user-attachments/assets/359c3120-aeb0-40f3-9771-e8ffe5c4dc59)

3. However, in the flow view the option is shown as undefined, and the options passed to the custom extension is an empty object.
![image](https://github.com/user-attachments/assets/bc5185b0-f9f0-47b0-a944-87652c93e85e)

### Directus Version

v10.12.0

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by br41nslug** [comment]
I agree that for such operations we should not have to inject the default values again. I believe this was inherited from how regular collections work, there the default value is handled by the database and purposefully not sent in the json payload.

**Comment by cany748** [comment]
This is true not only for custom operations, but also for internal ones. "exec" e.g. when saving with default value, it gives this error:
```
{
  "name": "TypeError",
  "message": "`eval` requires parameter 1 to be a string"
}
```

**Comment by julbd** [comment]
@danybol Did you find a workaround ?

**Comment by danybol** [comment]
@julbd in the end I just set the defaults in the extension code (i.e. if the option is undefined then set it to be whatever default I want). Not pretty but it works!
